lib/platform/gitlab/index.ts

```diff
@@ -370,10 +373,13 @@ function massagePr(prToModify: Pr): Pr {
 }
 
 async function fetchPrList(): Promise<Pr[]> {
-  const query = new URLSearchParams({
+  const searchParams = {
     per_page: '100',
-    author_id: `${authorId}`,
-  }).toString();
+  } as any;
+  if (!config.ignorePrAuthor) {
+    searchParams.author_id = `${authorId}`;
+  }
```

Comment:
```suggestion
  if (config.ignorePrAuthor) {
    // https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests
    // default: `scope=created_by_me`
    searchParams.scope = 'all';
  }
```

https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests